optimizely/helpers/condition.py

```diff
@@ -12,115 +12,180 @@
 # limitations under the License.
 
 import json
+import numbers
 
+from six import string_types
 
-class ConditionalOperatorTypes(object):
+from .validator import is_finite_number, are_values_same_type
```

Comment:
nit. I personally prefer doing something like:

`from . import validator` and then call the respective method.